src/base/static/components/place-detail/index.js

```diff
@@ -356,7 +360,11 @@ class PlaceDetail extends Component {
             isHorizontalLayout={isStoryChapter}
             numSupports={supports.length}
             onSocialShare={service =>
-              Util.onSocialShare(this.props.focusedPlace, service)
+              Util.onSocialShare({
```

Comment:
Thanks for fixing this! Note that we have an updated util here: https://github.com/mapseed/platform/blob/e9a3f57fb9a629f95bf9b379aa915dc4da072d98/src/base/static/utils/share-place.js#L20

I think we are trying to phase out the old `static/js/utils` file.

Reply:
Ah-- thanks for the heads up. I missed that there was a separate sharing util. I'll update this at some point.

Reply:
It's my bad - I should have made it clear that we were moving to another util for that. I should have left a comment.